lib/tasks/deployment/20201005203539_populate_case_contact_with_defaults.rake

```diff
@@ -0,0 +1,50 @@
+namespace :after_party do
+  desc 'Deployment task: populate_case_contact_with_defaults'
+  task populate_case_contact_with_defaults: :environment do
+    puts "Running deploy task 'populate_case_contact_with_defaults'"
+
+    CasaOrg.all.each do |org|
+      ContactTypeGroup.find_by(casa_org: org, name: "Biological family").update(name: "Family")
```

Comment:
Please move this setting of the name to the `populate_contact_type_groups_and_contact_types` task.